runtime/testapps/src/main/java/com/google/apphosting/runtime/jetty9/senderrorapp/unhandled-error.html 

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
The file name contains a space at the end.

Reply:
Fixed